sdk/attestation/azure-security-attestation/samples/README.md

```diff
@@ -170,4 +170,4 @@ This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For m
 [coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
 [coc_contact]: mailto:opencode@microsoft.com
 [readme_md]: https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/attestation/azure-security-attestation/README.md
-[add_isolated_mode_cert]https://azuresdkdocs.blob.core.windows.net/$web/cpp/azure-security-attestation/1.0.0-beta.1/class_azure_1_1_security_1_1_attestation_1_1_attestation_administration_client.html#ad5e9fed7924e3fa89a6b23ba12c5cb35
+[add_isolated_mode_cert]: https://azuresdkdocs.blob.core.windows.net/$web/cpp/azure-security-attestation/1.0.0-beta.1/class_azure_1_1_security_1_1_attestation_1_1_attestation_administration_client.html#ad5e9fed7924e3fa89a6b23ba12c5cb35
```

Comment:
The link here is out-dated and points to a doc version that is quite old (1.0.0-beta.1, before even 1.0.0 shipped).
Rather than hard-coding the version string, we should be using `latest`.

https://azuresdkdocs.blob.core.windows.net/$web/cpp/azure-security-attestation/latest/class_azure_1_1_security_1_1_attestation_1_1_attestation_administration_client.html#a65132a4cd2ce171286086650cec4964f

The out-dated link points to `AddPolicyManagementCertificate()` which doesn't match what we want.